examples/stage/odc_k8s/odc_k8s.tf

```diff
@@ -10,29 +10,42 @@ data "terraform_remote_state" "odc_eks-stage" {
 data "aws_caller_identity" "current" {
 }
 
+# collecting data from odc_eks module
+locals {
+  region       = data.terraform_remote_state.odc_eks-stage.outputs.region
+  owner        = data.terraform_remote_state.odc_eks-stage.outputs.owner
+  cluster_name = data.terraform_remote_state.odc_eks-stage.outputs.cluster_id
+
+  db_hostname       = data.terraform_remote_state.odc_eks-stage.outputs.db_hostname
+  db_admin_username = data.terraform_remote_state.odc_eks-stage.outputs.db_admin_username
+  db_admin_password = data.terraform_remote_state.odc_eks-stage.outputs.db_admin_username
+
+  node_role_arn = data.terraform_remote_state.odc_eks-stage.outputs.node_role_arn
+  user_role_arn = data.terraform_remote_state.odc_eks-stage.outputs.user_role_arn
+}
+
 module "odc_k8s" {
 //    source = "github.com/opendatacube/datacube-k8s-eks//odc_k8s?ref=terraform-aws-odc"
   source = "../../../odc_k8s"
   # Cluster config
-  region = data.terraform_remote_state.odc_eks-stage.outputs.region
-
-  owner = data.terraform_remote_state.odc_eks-stage.outputs.owner
-  cluster_name = data.terraform_remote_state.odc_eks-stage.outputs.cluster_id
+  region       = local.region
+  owner        = local.owner
+  cluster_name = local.cluster_name
 
   users = {
```

Comment:
Note during our debugging session earlier today the suggestion was to delete users if not used - but it doesn't have a default and is required so it needs a default added. Just noting this here whilst I remember since this PR is mucking with the same file.

Reply:
Thanks. Added defaults for users so it becomes optional now 👍